fuzzers/geojson_import_fuzzer.cpp

```diff
@@ -0,0 +1,75 @@
+/******************************************************************************
+ *
+ * Project:  PostGIS
+ * Purpose:  Fuzzer
+ * Author:   Even Rouault, even.rouault at spatialys.com
```

Comment:
Late to the party, but I guess this should be 2019 + your name, or the generic Postgis header right?

Reply:
Same for the other files